src/main/java/dev/meyi/bn/utilities/Utils.java

```diff
@@ -70,40 +79,51 @@ public static JSONArray unlockedRecipes() throws IOException {
 
       String _results = IOUtils
           .toString(new BufferedReader(new InputStreamReader(response.getEntity().getContent())));
-      JSONObject results = new JSONObject(_results);
+      JsonObject results = new JsonParser().parse(_results).getAsJsonObject();
```

Comment:
same thing here, we don't need to use a JsonParser and we certainly don't need to create a new one every single time you want to parse some json.